errormessages/src/ts/main.tsx

```diff
@@ -0,0 +1,110 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+/// <reference path="../tsd/tsd.d.ts" />
+
+import * as cu from 'cu-core';
+import events from 'cu-events';
+import * as React from 'react/addons';
+
+declare const cuAPI: any;
+
+class ErrorMessage extends React.Component<any, any> {
```

Comment:
We do not want explicit 'any' for Component property and state.  It's a bit more work to have an empty class, but it helps keep things type safe and future proof.  So just create a simple ErrorMessageState and ErrorMessageProps class that can be used in the template parameters for React.Component.